src/OSWindow-SDL2/OSSDL2Driver.class.st

```diff
@@ -361,33 +360,54 @@ OSSDL2Driver >> setGLAttributes: glAttributes [
 
 { #category : #'events-processing' }
 OSSDL2Driver >> setupEventLoop [
+
 	EventLoopProcess ifNotNil: [ 
-		EventLoopProcess isTerminating ifFalse: [EventLoopProcess terminate. EventLoopProcess := nil] ].
-	EventLoopProcess := self class isVMDisplayUsingSDL2 ifTrue: [ 
-		[ self eventLoopProcessWithVMWindow ] 
-		forkAt: Processor lowIOPriority
-	] ifFalse: [ 
-		self class hasPlugin ifTrue: [ 
-			[ self eventLoopProcessWithPlugin ] 
-			forkAt: Processor lowIOPriority ]
+		EventLoopProcess isTerminating ifFalse: [ 
+			EventLoopProcess terminate. 
+			EventLoopProcess := nil ] ].
+	
+	EventLoopProcess := self class isVMDisplayUsingSDL2 
+		ifTrue: [ 
+			[ self eventLoopProcessWithVMWindow ] 
+			forkAt: Processor lowIOPriority ] 
 		ifFalse: [ 
-			[ self eventLoopProcessWithoutPlugin ] 
-			forkAt: Processor lowIOPriority ].
-	].
+			self class hasPlugin 
+				ifTrue: [ 
+					[ self eventLoopProcessWithPlugin ] 
+					forkAt: Processor lowIOPriority ]
+				ifFalse: [ 
```

Comment:
I wonder what's the history of this plugin. In P10 and 11, at least, I only got false from `OSSDL2Driver hasPlugin`.